docs/cloudfoundry/cf-per-route-virtuals.rst

```diff
@@ -0,0 +1,156 @@
+----------------------------------------------------
+
+#. Create an Application Manifest.
+
+#. Include the :code:`bigip.status` and :code:`SERVICE_BROKER_CONFIG` sections of the Application Manifest, shown in the example below. The Service Plan, in the :code:`SERVICE_BROKER_CONFIG` section, contains the settings you want to apply to the virtual server. The :code:`f5servicebroker` Service will discover the Service Plan automatically.
+
+.. sidebar:: :fonticon:`fa fa-question-circle-o` Did you know?
+
+   F5 provides examples of custom local traffic policies in its BIG-IP documentation.
+
+   View the `PreventSpoofOfXFF policy`_ used in the example.
+
+You can attach any of the following BIG-IP objects to the virtual server:
+
+- policies (`BIG-IP Application Security Manager`_, L7 forwarding, compression, etc.),
+- profiles (tcp optimizations, x-forwarded-for, OneConnect, etc.),
+- pool settings (including **health monitors** and **load balancing algorithm**), and
+- server ssl profiles.
+
+The |cfctlr| cannot create or manage BIG-IP policies or profiles. With the exception of health monitors, all of the BIG-IP objects that you want to apply to the virtual server must already exist on the BIG-IP device.
+
+You can create a new health monitor in the Service Plan, use an existing BIG-IP health monitor, or both (shown in the example below).
+
+.. literalinclude:: /cloudfoundry/config_examples/manifest.yaml
+   :linenos:
+   :caption: Example App Manifest for cf-bigip-ctlr Service Broker
+
+:fonticon:`fa fa-download` :download:`Download the example manifest </cloudfoundry/config_examples/manifest.yaml>`
+
+.. _add-remove service plans:
+
+Add/Remove Service Plans
+````````````````````````
+
+You can use as many Service Plans as you need to define the BIG-IP services your Apps require. If you want to add or remove a Service Plan for a Controller that's already registered as a Service Broker:
+
+#. Edit the Application Manifest to add or remove the desired Service Plan.
+#. `Restart the App`_ to make the new settings take effect.
+
+When you remove a Service Plan from the Application Manifest, the Controller removes all of the BIG-IP objects associated with the Plan. This is the case regardless of whether any Routes are still bound to the Service when you delete the plan.
+
+.. note::
+
+   - If you remove all bound Routes before you delete the Plan, the Controller will clean up the BIG-IP objects and remove the Plan from the data group before the restart occurs.
+
+   - If you remove the Plan without unbinding the Routes, the Controller logs will show a diff of the data group vs any incoming Plans. It will remove all BIG-IP objects associated with the deleted plan. If you try to unbind a Route *after* removing its associated Plan, the Controller takes no action on the BIG-IP.
+
+   - Altering Plans that are already in effect may cause interruptions in traffic, depending on what settings have changed.
+
```

Comment:
I think we should add an additional note.

- Cloud Foundry only supports one route service binding per route. Plans should be designed to encompass the entire custom application configurations as application owners are restricted to a single binding.